dts/bindings/sensor/st,iis2dlpc-common.yaml

```diff
@@ -44,3 +44,76 @@ properties:
         - 2 # Low Power M3
         - 3 # Low Power M4
         - 4 # High Performance
+
+    # tap and tap-tap configuration section
+    # All default values are selected to match the power-up values.
+    # tap and tap-tap events can be generated on INT1 only.
+
+    tap-mode:
+      type: int
+      required: false
+      default: 0
+      description: Tap mode. Default is power-up configuration.
+      enum:
+        - 0 # Only Single Tap
+        - 1 # Single and Double Tap
+
+    tap-thsx:
```

Comment:
Based on other recent discussions, how about making this type `array` and documenting that it has three values corresponding to the X, Y, and Z threshold settings, and noting that a value of 0 means the axis is disabled.

Same for any other spatial properties (though I don't see any right now).

Reply:
OK, I'm acking also this one. Seems easy and it simplifies the code.